tests/foreman/ui/test_host.py

```diff
@@ -1907,13 +1907,15 @@ def gce_resource_with_image(
         # creating GCE CR from UI
         cr_name = gen_string('alpha')
         vm_user = gen_string('alpha')
+
+        with open(gce_cert['local_path']) as f:
+            json_key_content = f.read()
+
         session.computeresource.create(
             {
                 'name': cr_name,
                 'provider': FOREMAN_PROVIDERS['google'],
-                'provider_content.google_project_id': gce_cert['project_id'],
-                'provider_content.client_email': gce_cert['client_email'],
-                'provider_content.certificate_path': settings.gce.cert_path,
+                'provider_content.json_key': json_key_content,
```

Comment:
@adarshdubey-star has a PR https://github.com/SatelliteQE/robottelo/pull/10662 open for fixing this and moving this test to the GCE module

Reply:
Ok, but other changes I made can be merged or not? He will just do a little rebase then.

Reply:
You can revert the changes for this file @LadislavVasina1, the rest I think @shweta83 can answer as she is working on a similar fix.

Reply:
reverted